k8s_handle/__init__.py

```diff
@@ -106,9 +107,11 @@ def _handler_provision(command, resources, priority_evaluator, use_kubeconfig, s
 
     try:
         deprecation_checker = ApiDeprecationChecker(client.VersionApi().get_code().git_version[1:])
+        available_checker = ResourceAvailabilityChecker(make_resource_getters_list())
 
         for resource in resources:
             deprecation_checker.run(resource)
+            available_checker.run(resource)
```

Comment:
Any options to disable it? What if user don't have permissions to run this kind of API path?

Reply:
Today I checked whether I can restrict access for this API (api/{version} or /apis/{group}/version). 
Created token with access to get only Pod, but was still able to get a list of available versions.

Can you tell me, how set limitation for this API method? @seleznev @furiousassault

Reply:
Ok, `api` and `apis` available w/o auth as I see.